src/js/background/background.js

```diff
@@ -38,12 +39,27 @@ async function getAliasesFromServer(method = "GET", body = null, opts=null) {
     mode: "same-origin",
     method,
     headers: headers,
-    body,
   });
 
-  answer = await response.json();
+  const answer = await response.json();
+  const masks = new Array();
+  masks.push(...answer);
+
+  // If the user has domain (custom) masks set, also grab them before sorting
+  if (opts.subdomainSet) {
+    const domainResponse = await fetch(apiMakeDomainAddressesURL, {
+      mode: "same-origin",
+      method,
+      headers: headers,
+    });
+
+    const domainMasks = await domainResponse.json();
+    masks.push(...domainMasks);
+  }
   
-  return answer;
+  masks.sort((a, b) => (a.created_at < b.created_at ? 1 : -1));
```

Comment:
This actually didn't work when I swapped it in. Not sure on the logic here. I left it alone as it's working for me now. Happy to revisit!

Reply:
Ah OK, debugging locally I see why: `created_at` isn't actually a timestamp, but an ISO 8601 date string, e.g. `2021-11-23T12:55:22.630861Z`. And the reason the current code works, is because higher numbers have higher unicode code points (I assume), and when using `<` to compare strings, [its unicode code points are compared](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Less_than#description).

So it works, and should continue to work, like this, but it does feel a bit like magic, and can throw off future readers. We could consider throwing them into a `Date.parse()` (with the caveat that that's somewhat underspecified, but AFAICT works reliably across browsers for ISO 8601 strings) and then comparing `.getTime()`, but of course, that does only make this _more code_, so up to you.

Reply:
Yes! Sorry. I should've added more context in my initial comment. ISO 8601 is [lexicographically](https://en.wikipedia.org/wiki/Lexicographic_order) sortable, so my comparison works for now.